src/scale/ScaleManager.js

```diff
@@ -693,14 +693,19 @@ var ScaleManager = new Class({
      */
     lockOrientation: function (orientation)
     {
-        var lock = screen.lockOrientation || screen.mozLockOrientation || screen.msLockOrientation;
+        var lock = screen.lockOrientation || screen.msLockOrientation;
+        var locked = false;
 
         if (lock)
         {
-            return lock(orientation);
```

Comment:
I think changing this statement only would fix it:

```
return lock.call(screen, orientation);
```

Reply:
Ah ok, makes sense. I will try it and update the PR. Thanks @samme

Reply:
I made the above suggested change and tested it. It works well. I have created a PR with those changes. I didn't know how to squash the two commits without destroying this PR, so there are two commits here :sweat_smile: .

Reply:
I found out how to squash the commits. It didnt destroy the PR/